3rd-party/vaultpress.php

```diff
@@ -49,9 +52,10 @@ function jetpack_vaultpress_rewind_enabled_notice() {
  * @since 5.8
  */
 function jetpack_vaultpress_rewind_check() {
-	if ( Jetpack::is_active() &&
-		 Jetpack::is_plugin_active( 'vaultpress/vaultpress.php' ) &&
-		 Jetpack::is_rewind_enabled()
+	if (
+		Jetpack::is_active() &&
+		Jetpack::is_plugin_active( 'vaultpress/vaultpress.php' ) &&
+		Jetpack::is_rewind_enabled()
 		) {
```

Comment:
Feel free to reject, but I find it easier to read like this
```suggestion
	if (
		Jetpack::is_active() &&
		Jetpack::is_plugin_active( 'vaultpress/vaultpress.php' ) &&
		Jetpack::is_rewind_enabled()
	) {
```